java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -269,4 +271,51 @@ CompletableFuture<Boolean> pexpireAt(
      * }</pre>
      */
     CompletableFuture<Long> ttl(String key);
+
+    /**
+     * Invokes a Lua script with its keys and arguments. <br>
+     * This method simplifies the process of invoking scripts on a Redis server by using an object
+     * that represents a Lua script. The script loading, argument preparation, and execution will all
```

Comment:
```suggestion
     * that represents a Lua script. The script loading and execution will all
```
This function does not accept script arguments